ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -966,6 +966,22 @@ let migrate_send'  ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~options =
       debug "Mirror failed for VDI: %s" failed_vdi;
       raise (Api_errors.Server_error(Api_errors.mirror_failed,[Ref.string_of vconf.vdi]))
     end;
+
+    if (TaskHelper.is_cancelling ~__context) && not is_intra_pool then
```

Comment:
You are using parentheses for the `TaskHelper.is_canceling` function application but not for `not`. You don't need parentheses in either case (it's fine to write `if f x && not y then .. else ..`) but I think this should be consistent.